doc/releases/changelog-0.39.0.md

```diff
@@ -4,220 +4,384 @@
+  c1 = qml.noise.wires_in([0, 2]) 
+  n1 = qml.noise.partial_wires(qml.RY, -0.42)
+
+  noise_model = qml.NoiseModel({c1: n1}, meas_map={c0: n0})
+  ```
+
+  ```pycon
+  >>> noise_model
+  NoiseModel({
+    WiresIn([0, 2]): RY(phi=-0.42)
+  },
+  meas_map = {
+      MeasEq(expval): PhaseFlip(p=0.2)
+  })
+  ```
+
+  `qml.noise.meas_eq` can also be combined with other Boolean functions in `qml.noise` via bitwise operators
+  for more versatility.
+
+  To add this `noise_model` to a circuit, use the `qml.add_noise` transform as per usual. For example, 
+
+  ```python
+  @qml.qnode(qml.device("default.mixed", wires=3))
+  def circuit():
+      qml.RX(0.1967, wires=0)
+      for i in range(3):
+          qml.Hadamard(i)
+
+      return qml.expval(qml.X(0) @ qml.X(1))
+  ```
+
+  ```pycon
+  >>> noisy_circuit = qml.add_noise(circuit, noise_model)
+  >>> print(qml.draw(noisy_circuit)())
+  0: ──RX(0.20)──RY(-0.42)────────H──RY(-0.42)──PhaseFlip(0.20)─┤ ╭<X@X>
+  1: ──H─────────PhaseFlip(0.20)────────────────────────────────┤ ╰<X@X>
+  2: ──H─────────RY(-0.42)──────────────────────────────────────┤    
+  >>> print(circuit(), noisy_circuit())
+  0.9807168489852615 0.35305806563469433
+  ```
+
 <h4>User-friendly decompositions 📠</h4>
 
 * `qml.transforms.decompose` is added for stepping through decompositions to a target gate set. 
   [(#6334)](https://github.com/PennyLaneAI/pennylane/pull/6334)
 
 <h3>Improvements 🛠</h3>
 
+<h4>qjit/jit Compatibility and improvements</h4>
+
+* `qml.AmplitudeAmplification` is now compatible with qjit.
+  [(#6306)](https://github.com/PennyLaneAI/pennylane/pull/6306)
+
+* The quantum arithmetic templates are now compatible with qjit.
+  [(#6307)](https://github.com/PennyLaneAI/pennylane/pull/6307)
+  
+* The `qml.Qubitization` template is now compatible with qjit.
+  [(#6305)](https://github.com/PennyLaneAI/pennylane/pull/6305)
+
+* All PL templates are now unit tested to ensure jit compatibility.
+  [(#6309)](https://github.com/PennyLaneAI/pennylane/pull/6309)
+
+* The `qml.FABLE` template now returns the correct value when jit is enabled.
+  [(#6263)](https://github.com/PennyLaneAI/pennylane/pull/6263)
+
+* `qml.QutritBasisStatePreparation` is now jit compatible.
+  [(#6308)](https://github.com/PennyLaneAI/pennylane/pull/6308)
+
+* The `SampleMP.process_samples` method has been updated to support using JAX tracers for samples, allowing 
+  compatiblity with qjit workflows.
+  [(#6211)](https://github.com/PennyLaneAI/pennylane/pull/6211)
+
 <h4>Capturing and representing hybrid programs</h4>
 
-* `qml.wires.Wires` now accepts JAX arrays as input. Furthermore, a `FutureWarning` is no longer raised in `JAX 0.4.30+`
-  when providing JAX tracers as input to `qml.wires.Wires`.
+* `qml.wires.Wires` now accepts JAX arrays as input. Furthermore, a `FutureWarning` is no longer raised 
+  in `JAX 0.4.30+` when providing JAX tracers as input to `qml.wires.Wires`.
   [(#6312)](https://github.com/PennyLaneAI/pennylane/pull/6312)
```

Comment:
Some of these could be worth rewriting slightly to emphasize what the improvement is from a user point of view 🤔

Reply:
```suggestion
* `qml.wires.Wires` now accepts JAX arrays as input. Furthermore, a `FutureWarning` is no longer raised 
  in `JAX 0.4.30+` when providing JAX tracers as input to `qml.wires.Wires`.
  [(#6312)](https://github.com/PennyLaneAI/pennylane/pull/6312)
```

```suggestion
* `qml.wires.Wires` now accepts JAX arrays as input. In many workflows with JAX, PennyLane may attempt 
  to assign a JAX array to a `Wires` object, which will cause an error since JAX arrays are not hashable. Now, 
  JAX arrays are valid `Wires` types. Furthermore, a `FutureWarning` is no longer raised in `JAX 0.4.30+` 
  when providing JAX tracers as input to `qml.wires.Wires`.
  [(#6312)](https://github.com/PennyLaneAI/pennylane/pull/6312)
```